src/modules/layouts/SiteContext.js

```diff
@@ -42,6 +42,7 @@ const SiteProvider = ({ children, value }) => {
   `)
 
   const AllFilters = {}
+  const ExistsFilters = [{fragment: "Games", regex: /\[([^\[]+)\](\(.*\))/gm}]
```

Comment:
Ahh this is testing for a markdown-formatted link like `[stuff](url)`? Nice! Would you mind adding a quick comment or extracting it into a `markdownLinkRegex` variable for clarity/people like me? 😂  the joys of regex